src/main/java/org/zalando/nakadi/config/ValidatorConfig.java

```diff
@@ -65,6 +66,7 @@ public SchemaEvolutionService schemaEvolutionService() throws IOException {
         final Schema metaSchema = SchemaLoader.load(metaSchemaJson);
 
         final List<SchemaEvolutionConstraint> schemaEvolutionConstraints = Lists.newArrayList(
+                new CategoryChangeConstraint(),
```

Comment:
From my point of view categories relate to event type but not to the schema. I see it is handy to check it that way but what do you think about moving this out of schema validation classes?